spec/controllers/hyrax/generic_works_controller_spec.rb

```diff
@@ -199,18 +199,34 @@
       end
       let(:work) { instance_double(GenericWork, id: '99999', to_global_id: '99999') }
 
-      context 'with a user lacking workflow permission' do
+      context 'with a user lacking both workflow permission and read access' do
         before do
           allow(SolrDocument).to receive(:find).and_return(document)
+          allow(controller.current_ability).to receive(:can?).with(any_args).and_return(false)
```

Comment:
How about making this `.with(:read, SolrDocument)`?  I worry that `any_args` is too broad a scope and might bite us later since the allow is for the whole get request and not just a call to `document_not_found!`

Reply:
:+1: